parser.py

```diff
@@ -0,0 +1,34 @@
+import bs4
+import requests
+
+# вставить адрес с сайта лостфильм.тв, раздел гид по сериям
+httpAddress = 'https://www.lostfilm.tv/series/Rome/seasons'
+
+# название сериала из адреса
+seriesName = ((str(httpAddress)).split('/')[-2])
+if '_' in seriesName:
+    newName = seriesName.split('_')
+    print(*newName)
+else:
+    print(seriesName)
+print()
+
+# делаю суп с необходимыми данными из страницы
+httpFile = requests.get(httpAddress)
+soup = bs4.BeautifulSoup(httpFile.text, 'html.parser')
+dataSoup = soup.findAll(attrs={"class": "details"})
+
+# количество сезонов
+amountSeason = len(dataSoup)
+
+# год выхода сезона и количество эпизодов из супа
+i = 0
+for data in dataSoup:
```

Comment:
Если нужно получить и элемент массива и его индекс более правильно использовать функцию `enumerate`
```suggestion
for i, data in enumerate(dataSoup):
```
Тогда и строчку выше и увеличение счетчика на 1 можно удалить